src/WebJobs.Extensions.DocumentDB/Config/DocumentDBConfiguration.cs

```diff
@@ -46,6 +46,11 @@ public DocumentDBConfiguration()
         /// </summary>
         public ChangeFeedHostOptions LeaseOptions { get; set; }
 
+        /// <summary>
+        /// Gets or sets the ConnectionMode used in the DocumentClient instances.
+        /// </summary>
+        public string ConnectionMode { get; set; }
```

Comment:
Chatted with @fabiocav. Let's have this be the `ConnectionMode` enum instead of string. Then we can handle the parsing and conversion here; https://github.com/Azure/azure-functions-host/blob/v1.x/src/WebJobs.Script/Binding/DocumentDBScriptBindingProvider.cs#L47

Reply:
Got it, will update the PR during the day.